### PR TITLE
remove some unused functions in client-go(tools/clientcmd)

### PR DESCRIPTION
**Pull request by NickrenREN**
@caesarxuchao  I removed some unused function. PTAL. Thanks

### CODE REVIEW

File: staging/src/k8s.io/client-go/tools/clientcmd/loader.go [hunk 1]
```diff
@@ -528,27 +527,6 @@ func RelativizeAuthInfoLocalPaths(authInfo *clientcmdapi.AuthInfo) error {
 	return nil
 }
 
-func RelativizeConfigPaths(config *clientcmdapi.Config, base string) error {
```
Comment (deads2k) [inline staging/src/k8s.io/client-go/tools/clientcmd/loader.go]: These ought to stay.  Also, if no one is using them, then something is probably going wrong.

Comment (caesarxuchao) [inline staging/src/k8s.io/client-go/tools/clientcmd/loader.go]: These are public function, we don't know if there are users depending on them.

Comment (NickrenREN) [inline staging/src/k8s.io/client-go/tools/clientcmd/loader.go]: I can not find where they are used. Anyway, sorry for wasting your time

Comment (caesarxuchao) [inline staging/src/k8s.io/client-go/tools/clientcmd/loader.go]: By "users" I mean whoeever imports client-go in their own project, not necessarily in k8s.io/kuberentes.

Comment (NickrenREN) [inline staging/src/k8s.io/client-go/tools/clientcmd/loader.go]: Oh,I see. Thanks

**Pull request closed without merging**